### PR TITLE
Update to k8s client-go 1.20

### PR DESCRIPTION
**Pull request by adityadani**
**What type of PR is this?**
cleanup

**What this PR does / why we need it**:
Updates client-go to 1.20

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:

```release-note
Updated k8s client-go to 1.20
```

**Does this change need to be cherry-picked to a release branch?**:
No

### CODE REVIEW

File: pkg/extender/extender.go [hunk 1]
```diff
@@ -122,6 +122,9 @@ func (e *Extender) Stop() error {
 }
 
 func (e *Extender) serveHTTP(w http.ResponseWriter, req *http.Request) {
+	pvc1, err1 := core.Instance().GetPersistentVolumeClaim("restorePVC", "default")
```
Comment (ram-infrac) [inline pkg/extender/extender.go:125]: why is this code block needed here ?did you add it for debug purposes?

**Review by ggriffiths — APPROVED** [review]
Lgtm once the test debug line is removed.

Thanks for finishing this up! Maybe it's worth running a stork integration test with this update?

File: pkg/extender/extender.go (hunk 1 above)
Comment (adityadani) [reply pkg/extender/extender.go:125]: Yes I did for debugging a UT. Good catch. Will remove this.

**Comment by adityadani** [comment]
stork integration test passed - https://jenkins.portworx.dev/job/Stork/job/stork-2-dev-test/179/

**Review by ram-infrac — APPROVED** [review]

**Pull request merged**